domain/resourcegroup/runaway.go

```diff
@@ -104,6 +197,7 @@ func newRunawayChecker(manager *RunawayManager, resourceGroupName string, settin
 		deadline:          time.Now().Add(time.Duration(setting.Rule.ExecElapsedTimeMs) * time.Millisecond),
 		setting:           setting,
 		marked:            atomic.Bool{},
+		action:            strings.ToLower(setting.Action.String()),
```

Comment:
why not get it from `setting` when needed

Reply:
I want to reduce the use of `strings.ToLower`.